config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_storagebuckets.storage.cnrm.cloud.google.com.yaml

```diff
@@ -287,6 +287,23 @@ spec:
                 required:
                 - retentionPeriod
                 type: object
+              softDeletePolicy:
+                description: The bucket's soft delete policy, which defines the period
+                  of time that soft-deleted objects will be retained, and cannot be
+                  permanently deleted. If it is not provided, by default Google Cloud
+                  Storage sets this to default soft delete policy.
+                properties:
+                  effectiveTime:
```

Comment:
This really should be status (though it's not because it's nested)

Also, I think we want retentionDurationSeconds to be in both spec and status.observedState

And I think we want to be sure that we only write back to status.observedState (which I think is fair because it's a new field, nobody's going to be looking in the "old place").